src/sidebar/templates/sidebar-content.html

```diff
@@ -8,27 +8,21 @@
 </selection-tabs>
 
 <search-status-bar
-  ng-if="!vm.isLoading()">
+  ng-if="!vm.isLoading() && !(vm.selectedAnnotationUnavailable() || vm.selectedGroupUnavailable())">
```

Comment:
I'm hoping that when we migrate this component, we can make this logic less hokey.

Reply:
Can you clarify why this is needed?

Reply:
We're (as of these changes) not rendering the `SearchStatusBar` at all if there is a direct-linked error. Instead, we're rendering our own "Clear filter" (i.e. "Show all annotations") button within `SidebarContentError`. This allows a bit of simplification to the code in `SearchStatusBar` as well, as it no longer has to concern itself with direct-linked problems.